src/main/java/com/codegym/jrugotom5/exception/GlobalExceptionHandler.java

```diff
@@ -1,14 +1,22 @@
 package com.codegym.jrugotom5.exception;
 
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
 @ControllerAdvice
+@Slf4j
 public class GlobalExceptionHandler {
     @ExceptionHandler(InvalidDateRangeException.class)
     public ResponseEntity<String> handleInvalidDateRange(InvalidDateRangeException ex) {
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(UserNotFoundException.class)
+    public ResponseEntity<String> handleUserNotFound(UserNotFoundException ex) {
+        log.error("Exception occurred: {}", ex.getMessage());
```

Comment:
added a timestamp and a status code, if there is any more info an exception could give could you please inform me

Reply:
i suppose mentor means that you put not enough information in your LOG message (not information for client). And it relates to both of your exception handlers. It will be hard to debug application when you will have just only "Message of your exception"
see:
https://github.com/Volodymyr95/goto-m5f/pull/16#discussion_r1812546592
https://github.com/Volodymyr95/goto-m5f/pull/13#discussion_r1812556175